### DISCUSSION TITLE
Error updating Calendar app

### DISCUSSION THREAD

**Discussion by jeffstoner** [nextcloud/all-in-one#2716] [Questions] [answered]
### Steps to reproduce
1. Open Apps interface
2. Click Update button to update Calendar
3. 

### Expected behavior
Calendar app should be updated

### Actual behavior
This error is displayed in a toast:
`Database error when running migration 4004Date20230414163505 for app calendar Migration step 'OCA\Calendar\Migration\Version4004Date20230414163505' is unknown`

### Host OS
Ubuntu 22.04.2 LTS

#### Nextcloud AIO version
6.0.0

#### Current channel
latest

#### Other valuable info
The Contacts app also cannot be updated but no useful error message was displayed, only "An error occurred during the request. Unable to proceed."

I can no longer log in to NextCloud. I get the following message:
"Update needed
Please use the command line updater because automatic updating is disabled in the config.php.

For help, see the [documentation](https://docs.nextcloud.com/server/26/go.php?to=admin-cli-upgrade)."

Potentially related to #2712

**Comment by GUNT0815** [comment]
Same here:

- upgrade to Nextcloud-aio 6
- running for one day
- via admin user in nextcloud gui: app updates 
- unsuccessfull for calendar app with error message
- nextcloud webui not reachable afterwards, webpage message: "Aktualisierung erforderlich
Da automatische Updates in der config.php deaktivert sind, nutzen Sie bitte den Updater über die Kommandozeile.
Für weitere Hilfen, schauen Sie bitte in die [Dokumentation](https://docs.nextcloud.com/server/26/go.php?to=admin-cli-upgrade)."
- manual update try via commandline -> looks like successful
- remark: caldav is also not working

**Comment by Zoey2936** [comment, marked as answer]
can you please try: https://github.com/nextcloud/all-in-one/discussions/1787 ?

**Reply by Roman991** [reply]
it worked :)

**Reply by jeffstoner** [reply]
Thank you

**Comment by szaimen** [comment]
The bug is reported to https://github.com/nextcloud/calendar/issues/5191  and will be fixed with https://github.com/nextcloud/calendar/pull/5295

**Reply by jeffstoner** [reply]
Awesome!

**Comment by szaimen** [comment]
As for AIO see 

> can you please try: #1787 ?
